### PR TITLE
Feature/connect eoa wallet link

### PR DESCRIPTION
**Pull request by BellringerQuinn**
Added eoa wallet linking via url generation to SDK. Added a button in demo page for this as well.

Also added session auth proof generation to WaaSWallet

### CODE REVIEW

File: Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Sequence.WaaS
+{
+    public class EOAWalletLinker
+    {
+        private static string NonceGenerationLink =
+            "https://demo-waas-wallet-link-server.tpin.workers.dev/generateNonce";
```
Comment (taylanpince) [inline Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs:9]: Can you make this configurable? We shouldn't embed our demo server in the SDK. Then from Jelly Forest implementation you can pass this URL.

**Review by taylanpince — CHANGES_REQUESTED** [review]
Looks good just one detail needed.

File: Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs (hunk 1 above)
Comment (BellringerQuinn) [reply Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs]: Done, it is now passed in via the constructor.
Do we have a doc on how devs can configure one of these servers?

Comment (taylanpince) [reply Assets/Sequence/SequenceSDK/WaaS/EOAWalletLinker.cs]: Nice, thanks. Yep we have one here: https://docs.sequence.xyz/solutions/wallets/link-wallets/integration-guide

**Pull request merged**